docs/bokeh/source/docs/first_steps/first_steps_1.rst

```diff
@@ -54,21 +60,24 @@ right of the plot to explore:
 
 Follow these steps to recreate this simple line chart:
 
-1. Import the necessary functions from the |bokeh.plotting| module:
+1. Create a new Python file on your machine (e.g. simple_line_chart.py) and open it in a code editor of your choice (such as |Sublime Text|, 
```

Comment:
In RST, you can use double backticks (``) to format text as code, that's what I'd use for the example file name:

```suggestion
1. Create a new Python file on your machine (e.g. ``simple_line_chart.py``) and open it in a code editor of your choice (such as |Sublime Text|, 
```

Reply:
Also the references need to be like: 
```
`Sublime Text`_, `Visual Studio Code`_, etc.
```
The syntax with the pipes, e.g. `|bokeh.plotting|` is actually a replacement substitution, not a reference (Sphinx has lots of obscure syntax)

Reply:
Thanks for all the feedback!